### ISSUE TITLE
Parameterization is not working in Lavague-QA: Variable parameter is passed in to input field as it is, instead of values mentioned under "Examples" section of xxxx.feature file 

### ISSUE THREAD

**Issue by manikandanma17** [lavague-ai/LaVague#609]
**Describe the bug**
Pre-requisites: 

Below is the xxxxxx.feature file
Feature: Create Product

  Scenario Outline: Create a product
    Given Launch web app URL
    When User is on the login page
    And Maximize the page
    And Enter **"<username>"** into "Username" field
    And Enter **"<password>"** into "Password" field
    And Click on "SIGN IN" button

    Examples:

      | username | password |
      | test     | test123  |

**To Reproduce**
1. Run below CLI command 

lavague-qa --url https://xxxxxxx.com/ --feature feature/create_product.feature

2. Observe that the variable mentioned in **"Scenario Outline"** is passed into input field as it is, instead of passing the corresponding value mentioned under "**Examples**" section
<img width="477" alt="Screenshot 2024-09-30 at 3 08 42 PM" src="https://github.com/user-attachments/assets/e5f95458-e537-4636-bb46-3140b4adf541">
<img width="400" alt="Screenshot 2024-09-30 at 3 08 19 PM" src="https://github.com/user-attachments/assets/0cceaaf3-043d-4912-a1b6-7636f3896b91">

 Note: Screenshots and Logs are attached here for your reference.

**Expected behavior**

The value mentioned under **"Examples"** section should be passed into respective input field as per variable mentioned in **"Scenario Outline"** section

Logs:

  warnings.warn(warning_message, UserWarning)
Ready to generate tests on https://xxxxxxx.com/ for Feature: Create Product

  Scenario Outline: Create a product
    Given Launch web app URL
    When User is on the login page
    And Maximize the page
    And Enter "<username>" into "Username" field
    And Enter "<password>" into "Password" field
    And Click on "SIGN IN" button

    Examples:

      | username | password   |
      | test  | test123 |

2024-09-30 14:53:24,975 - INFO - Screenshot folder cleared
2024-09-30 14:53:34,002 - INFO - Thoughts:
- The current screenshot shows the login page for test-apps
- The objective is to ensure the user is on the login page, which is already achieved.
- The next step is to provide confirmation that the user is on the login page.

Next engine: COMPLETE
Instruction: The user is on the login page of test-apps.
2024-09-30 14:53:34,004 - INFO - Objective reached. Stopping...
2024-09-30 14:53:40,055 - INFO - Thoughts:
- The current screenshot shows a login page for test-apps.
- The objective is to maximize the page.
- Maximizing the page will provide a better view and potentially reveal more elements that are currently not visible.
- The best next step is to use the Navigation Controls to maximize the window.

Next engine: Navigation Controls
Instruction: MAXIMIZE_WINDOW
2024-09-30 14:53:49,271 - INFO - Thoughts:
- The current screenshot shows a login page for test-apps.
- The objective is to enter "<username>" into the "Username" field.
- The "Username" field is visible and ready for interaction.
- The next step should involve entering the provided username into the "Username" field.

Next engine: Navigation Engine
Instruction: Enter **"<username>"** into the "Username" field.
2024-09-30 14:54:05,978 - INFO - Thoughts:
- The current screenshot shows a login page for test-apps.
- The objective is to enter a password into the "Password" field.
- The "Password" field is visible and ready for input.
- The next step is to enter the given password into the "Password" field.

Next engine: Navigation Engine
Instruction: Enter **"<password>**" into the "Password" field.
2024-09-30 14:54:16,183 - INFO - ### Thoughts:
- The current screenshot shows the sign-in page for test-apps.
- The username and password fields have been filled in the previous steps.
- The next step is to click on the "SIGN IN" button to proceed with the login process.

<img width="477" alt="Screenshot 2024-09-30 at 3 08 42 PM" src="https://github.com/user-attachments/assets/e0011f9d-a5c4-4a11-9ae3-e9101f9fbadf">
<img width="400" alt="Screenshot 2024-09-30 at 3 08 19 PM" src="https://github.com/user-attachments/assets/80a0d27c-61f7-4ba6-b130-6518f1d38070">

**Comment by manikandanma17** [comment]
It is my humble request to Lavaque team to look in to this issue on high priority as i am blocked with Parameterization of data passing.

**Comment by dhuynh95** [comment]
Hi @manikandanma17,
We are working on a revamp from scratch that will make our Agent much more efficient. 
We will release it in a week or so and it should solve your issue.
In the meantime, could you either share more information on the website if it's public, and if it's not, could you ping me on my Discord to see how we can figure this out? My handle is DHuynh95

**Comment by manikandanma17** [comment]
@dhuynh95 Thanks for responding to my issue. The web app that i am using for testing is of our organization, hence i could not share it here. But the scenario which i am trying to handle is very simple and generic.

Say for example: 
1.Launch a web app which takes user to login page
2.Now, i want to Lavague-QA to read **test** and **test123** values which are stored under "Examples" section of **login_test.feature** file in to respective input box (username/password) 
3. Currently observing it is passing the "<username>" as it is, in to the input box instead of referring to the values mentioned below.

Below is the xxxxxx.feature file
Feature: Create Product

**Scenario Outline:** Create a product
Given Launch web app URL
When User is on the login page
And Maximize the page
And Enter "" into "Username" field
And Enter "" into "Password" field
And Click on "SIGN IN" button

**Examples:**

  | username | password |
  | test     | test123  |

**Comment by manikandanma17** [comment]
@dhuynh95 Could you please let me know once Lavague latest version is released? so that i will be able to check parameterization is working or not. 
Thanks.

**Comment by dhuynh95** [comment]
Hi @manikandanma17,
Sorry for the time to answer, big launch in preparation.
We are opening a private beta to test the new performance of our Agent API. Would you be interested in trying? If so, we can discuss together to see how we could onboard you.

**Comment by manikandanma17** [comment]
@dhuynh95 Yes, before that, Could you please tell me if anything got changed in the backend of Lavague already? The reason why i am asking is, I generated the scripts using below CLI commands and then I tried to run xxxxx.py file generated by Lavague-qa, which is throwing **selenium.common.exceptions.TimeoutException: Message:**

Note: This was working fine till last friday (4th Oct 2024), but start from today getting "Timeout exception"

lavague-qa --url "xxxxx.com" --feature /feature/xxxxx.feature
 pytest generated_tests/xxxxx.py
 
 If you dont mind, can we please connect on this issue?

**Comment by manikandanma17** [comment]
@dhuynh95  Thanks for the reply. Apologies, as I am quite occupied with some urgent work, I would not be able to take part of what you are asking. 

 Sorry if i am bothering you on same thing again, Could please let me know when latest Lavague build will be released? as I am currently blocked to perform any tests with parameterization passing in it.

**Comment by manikandanma17** [comment]
@dhuynh95 Could you please let me know if LaVague latest version is released already or not? As you have mentioned above in one of your reply that you are planning for big launch for LaVague.
I am eagerly waiting to see parameterisation issue is resolved with a new release of LaVague .

**Comment by dhuynh95** [comment]
Hi @manikandanma17,
Sorry for not providing news yet. We are finishing cooking and we have an initial version that looks good for testing!
We will release officially end of January but we want to onboard a small pool of early users to get quick feedback from a small group of people before releasing for everyone.
Would you be interested in early access? If so, you can ping me on Discord (dhuynh95 or dhuynh)

**Comment by manikandanma17** [comment]
Hi @dhuynh95  I am really sorry, I will not be able to participate in any early access/testing, reason being i am currently loaded with our organization release activities.

**Comment by manikandanma17** [comment]
@dhuynh95  Could you please let me know if latest version of LaVaque-QA is released with fixes for 
**Parameterization is not working in Lavague-QA: Variable parameter is passed in to input field** ?
